node/flatpak_node_generator/providers/npm.py

```diff
@@ -311,6 +311,8 @@ async def generate_package(self, package: Package) -> None:
             self.gen.add_git_source(source.url, source.commit, path)
 
         elif isinstance(source, LocalSource):
+            if Path('node_modules') in Path(source.path).parents:
```

Comment:
Hmm, should we...maybe just remove the assert altogether? This feels like a slightly odd heuristic to me (ngl although I'm not sure how it ended up in the lockfile in the first place)

Reply:
Well, ignoring local sources altogether doesn't feel right, but I guess we don't have anything to do with them anyway.